sdk/nodejs/src/client.ts

```diff
@@ -115,7 +124,7 @@ export class DevCycleClient {
                     platformVersion: process.version,
                     sdkType: 'server',
                     sdkVersion: packageJson.version,
-                    hostname: os.hostname(),
+                    hostname: this.hostname,
```

Comment:
this is targeting data, you can't change this to not be the hostname

Reply:
I don't think it is targeting data...

Reply:
Yea I don't see it being used as part of targeting data, only for reporting on events.

Reply:
I'm happy to have you double-check, but I only see this data used for the `user_id` of `agg` events and reported as part of the populated user object for all events.

Reply:
it's part of the automatically set user data though, I don't see why we should change the definition of hostname to not be the hostname

Reply:
As mentioned in the thread above, we want to try to align the user_id used by the `agg` events and this new `sdkConfig` event. I was trying to avoid making more changes to the bucketing lib, but I can separate the `uuid` from the `hostname` if needed. But it really didn't seem necessary, as the `hostname` is only ever used for reporting and not as part of any targeting rule.

Reply:
for now maybe, it seems like an unnecessary risk to me to change the meaning of that field, and it's kinda confusing to keep track of.